channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/dto/JaicpAnalyticsAPI.kt

```diff
@@ -83,6 +83,29 @@ class JaicpAnalyticsAPI internal constructor() {
         this.sessionResult = result
     }
 
+    override fun equals(other: Any?): Boolean {
+        if (this === other) return true
+        if (javaClass != other?.javaClass) return false
+        other as JaicpAnalyticsAPI
+
+        if (sessionResult != other.sessionResult) return false
+        if (comment != other.comment) return false
+        if (sessionData != other.sessionData) return false
+        if (sessionLabel != other.sessionLabel) return false
+        if (messageLabel != other.messageLabel) return false
+
+        return true
+    }
+
+    override fun hashCode(): Int {
```

Comment:
`Objects.hash(...)`?

Reply:
Using `data class` still was the best option)